src/handler/sign_tx.c

```diff
@@ -65,29 +65,46 @@ bool verify_change(change_output_info_t *info, tx_output_t output) {
  *
  * The outputs will be on the global context for sign tx (`tx_info`)
  **/
-void read_change_info_v1(buffer_t *cdata) {
+uint16_t read_change_info_v1(buffer_t *cdata) {
```

Comment:
chore: We need to update the docstrings with the return value and its meaning

Reply:
Done